tensorflow/docs_src/programmers_guide/debugger.md

```diff
@@ -795,7 +796,7 @@ sess.run(b)
        tensor `b` is effectively also a constant tensor. TensorFlow's graph
        optimization folds the graph that contains `a` and `b` into a single
        node to speed up future runs of the graph, which is why `tfdbg` does
-       not generate any intermediate tensor dumps. However, if `a` were a
+       not generate any intermediate tensor dumps. However, if `a` was a
```

Comment:
`were` is correct. You should use a past subjunctive (https://en.wikipedia.org/wiki/English_subjunctive#Use_of_the_past_subjunctive) here, since it describes a hypothetical scenario, and `were` is the past subjunctive form of `be`, regardless of whether the subject is plural.

Reply:
Thanks a lot, I've reverted this.